syntax = "proto3";

message Block {
    string title = 2;
    string description = 3;
    string link = 4;
    string blog_title = 5;
    string blog_link = 6;
    repeated string authors = 7;
    string time_published = 8;
}

message InputData {
    repeated Block block = 1;
}

message StatusCode {
    int32 status = 1;
    string message = 2;
}

service Train {
    rpc Training (InputData) returns (StatusCode) {};
}